l1-contracts/contracts/bridge/asset-router/L2AssetRouter.sol

```diff
@@ -126,6 +126,22 @@ contract L2AssetRouter is AssetRouterBase, IL2AssetRouter {
         emit DepositFinalizedAssetRouter(L1_CHAIN_ID, _assetId, _transferData);
     }
 
+    /*//////////////////////////////////////////////////////////////
+                     Internal & Helpers
+    //////////////////////////////////////////////////////////////*/
+
+    // kl todo add handle Legaacy data here, which calls esureTokenRegisteredWithNTV
```

Comment:
just in case, do we need to do it in this PR?

Reply:
We need it with Raid's fixes